playground/src/model/exercise.ts

```diff
@@ -7,7 +7,7 @@ type ExerciseBase = {
   type: ExerciseType;
   max_points: number;
   bonus_points: number;
-  grading_instructions: string;
+  grading_instructions: string | null;
```

Comment:
I like `undefined` more than `null`, idk what is really best practices but I always check for `undefined` instead of `null`. I guess semantically, `null` is more sensible but I still prefer `undefined`. 
```suggestion
  grading_instructions?: string;
```

Reply:
Makes sense! It's also the preferred choice in Artemis according to our guidelines, so we probably want to also do this 👍